frontend/lib/src/SessionInfo.ts

```diff
@@ -49,6 +49,20 @@ export class SessionInfo {
    */
   private _last?: Props
 
+  /**
+   * An ID uniquely identifying the last ForwardMsg that we received from the
+   * server.
+   */
+  private _lastForwardMsgID?: string
```

Comment:
nit: I don't know whether we follow a specific style guide, but the Google Typescript Styleguide [recommends](https://google.github.io/styleguide/tsguide.html#camel-case) to write this as `_lastForwardMsgId`

Reply:
Hm, I don't think we have anything official to determine how we name these types of things, although I think following the google style guide would be better since it also will be consistent with autogenerated proto names. Changed to `_lastForwardMsgId`